From 9194e1425aebcfd7b1d3bf218de4430853dc9e37 Mon Sep 17 00:00:00 2001
From: Brandon Philips <brandon@ifup.co>
Date: Mon, 4 Mar 2013 13:15:29 -0800
Subject: [PATCH] hack(Makefile.am): explicit -lpthread to fix pthread

for some reason pthread_LIBS isn't getting populated. I am not terribly
interested in hunting this down quite yet so hack it for now.

Change-Id: Id18dba7b2d1d9004478d3bc736436628c971f284
---
 src/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/Makefile.am b/src/Makefile.am
index 709ba19..ab7d7de 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -43,7 +43,7 @@ lib_include_HEADERS = $(HEADER_FILES)
 
 lib_LTLIBRARIES = libdbus-c++-1.la
 libdbus_c___1_la_SOURCES = $(HEADER_FILES) interface.cpp object.cpp introspection.cpp debug.cpp types.cpp connection.cpp connection_p.h property.cpp dispatcher.cpp dispatcher_p.h pendingcall.cpp pendingcall_p.h error.cpp internalerror.h message.cpp message_p.h server.cpp server_p.h eventloop.cpp eventloop-integration.cpp $(GLIB_CPP) $(ECORE_CPP)
-libdbus_c___1_la_LIBADD = $(dbus_LIBS) $(glib_LIBS) $(pthread_LIBS) $(ecore_LIBS)
+libdbus_c___1_la_LIBADD = -lpthread $(pthread_LIBS) $(dbus_LIBS) $(glib_LIBS) $(ecore_LIBS)
 
 MAINTAINERCLEANFILES = \
 	Makefile.in
-- 
1.8.1

